size_comparison/README.md

```diff
@@ -0,0 +1,151 @@
+# Futures versus no-Futures Size Comparison
+
+Original author: Johnathan Van Why
+
```

Comment:
Can you say what commit these numbers were generated for? That will let us keep track of the results.

Reply:
I added a commit hash. The numbers were actually slightly out of date -- while fixing an entry point bug I moved 1 word from .text into .rt_header. The write-up and disassembly now match the latest commit.